app/Http/Resources/IndexDoctorResource.php

```diff
@@ -1,22 +0,0 @@
-<?php
-
-namespace App\Http\Resources;
-
-use Illuminate\Http\Resources\Json\JsonResource;
-
-class IndexDoctorResource extends JsonResource
```

Comment:
Why those files are deleted?

Reply:
solved by git pull and solving conflicts